bees/cleverbotbee/README.md

```diff
@@ -1,11 +1,7 @@
-# Cleverbotbee
+# Cleverbot bee
 
-CleverbotBee is a simple bee chatting with cleverbot.
-You can send messages to the bot as action and you'll receive the answer as an
-event.
+CleverbotBee is a simple bee chatting with cleverbot. You can send messages to the bot as action and you'll receive the answer as an event.
```

Comment:
Might be worth linking the first instance of `CleverBot` (proper capitalization?) to the website / Git repo of CleverBot.